includes/definitions/discovery/ray.yaml

```diff
@@ -37,7 +37,7 @@ modules:
             data:
                 -
                     oid: ber
-                    num_oid: '.1.3.6.1.4.1.33555.4.3.2.11.{{ $index }}'
+                    num_oid: '.1.3.6.1.4.1.33555.1.3.2.11..{{ $index }}'
```

Comment:
You cannot have 2 dots in an OID. So this line must be wrong. Could you fix it  ?